File: eladmin/eladmin-sever/eladmin-system/src/main/java/me/zhengjie/modules/security/security/AuthFilter.java
package me.zhengjie.modules.security.security;

import lombok.extern.slf4j.Slf4j;
import org.springframework.stereotype.Component;

import javax.servlet.*;
import javax.servlet.http.HttpServletRequest;
import java.io.IOException;

/**
 * 这个类并没有什么作用,只是测试过滤器和拦截器的区别
 */
@Slf4j

public class AuthFilter implements Filter {
    @Override
    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
        log.debug("start to auth request validate...111");
        HttpServletRequest req = (HttpServletRequest) request;
        String token = req.getHeader("Authorization");
        if (token  == null) {
            //    :TODO check token
            log.info("auth success");

        } else {
            log.error("auth failed");
        }
        chain.doFilter(request, response);
    }
}

File: eladmin/eladmin-kt/eladmin-system/src/main/kotlin/me/zhengjie/modules/security/security/AuthFilter.kt
package me.zhengjie.modules.security.security

import lombok.extern.slf4j.Slf4j
import java.io.IOException
import javax.servlet.*
import javax.servlet.http.HttpServletRequest

/**
 * 这个类并没有什么作用,只是测试过滤器和拦截器的区别
 */
@Slf4j
class AuthFilter : Filter {
    @Throws(IOException::class, ServletException::class)
    override fun doFilter(request: ServletRequest, response: ServletResponse, chain: FilterChain) {
        AuthFilter.log.debug("start to auth request validate...111")
        val req = request as HttpServletRequest
        val token = req.getHeader("Authorization")
        if (token == null) {
            //    :TODO check token
            AuthFilter.log.info("auth success")
        } else {
            AuthFilter.log.error("auth failed")
        }
        chain.doFilter(request, response)
    }
}